python/llama-index-server/llama_index/server/services/workflow.py

```
@@ -0,0 +1,122 @@
+
+    @staticmethod
+    def get_storage_path(id: str) -> Path:
+        storage_dir = Path("output") / "checkpoints"
+        if not storage_dir.exists():
+            storage_dir.mkdir(parents=True, exist_ok=True)
+        return storage_dir / f"{id}.json"
+
+    @classmethod
+    async def save_context(
+        cls,
+        id: str,
```

Pylint (3.3.7)

[warning] Redefining built-in 'id'

(W0622)